clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -6561,18 +6561,28 @@ def ext_goto_into_protected_scope : ExtWarn<
 def warn_cxx98_compat_goto_into_protected_scope : Warning<
   "jump from this goto statement to its label is incompatible with C++98">,
   InGroup<CXX98Compat>, DefaultIgnore;
+def warn_cpp_compat_goto_into_protected_scope : Warning<
```

Comment:
It looks like this diagnostic and the last one are not covered in the test.

Reply:
This one is covered by:
```
void goto_func_1(void) {
  goto ouch;  // c-warning {{jump from this goto statement to its label is incompatible with C++}} \
                 cxx-error {{cannot jump from this goto statement to its label}}
  int i = 12; // both-note {{jump bypasses variable initialization}}

ouch:
  ;
}
```
But the computed goto one is not covered yet.